docs/adr/0039-use-react-container-component-abstraction.md

```diff
@@ -0,0 +1,94 @@
+# Use React Container Component Abstraction
+
+- Status: accepted
+- Deciders: Lauren Zugai, Dan Schomburg
```

Comment:
@chenba If you get a chance to look this over I'm going to add your name here if that's cool since you gave some feedback on the prototoype.